sofkasim.py

```diff
@@ -94,8 +95,10 @@ def out(self, history=True):
         L1 = min(1, self.durations[1] / self.durations[0])
         on = self.durations[1] - attack
         decay = min(self.durations[2], self.durations[3])
-        L3 = L1 * (1 - (decay / self.durations[2]))
+        L3 = L1 * (1 - (decay / self.durations[2])) if decay else 1
         dur = round(attack + on + decay, 3)
+        if not decay:
+            on -= 0.02
```

Comment:
should add comment to explain this , from Nyquist manual:

> If t1 + t2 + 2ms + t4 is greater than the envelope duration, then a two-phase envelope is substituted that has an attack/release time ratio of t1/t4.

We don't want a 2 phase env here.

Reply:
latest #10 now uses `(pwl-list)` which has different behaviour. This comment only applied to the original `(env)` function.